tests/conftest.py

```diff
@@ -671,6 +684,18 @@ def unauthenticated_find_test_client() -> Generator[FlaskClient, None, None]:
         yield test_client
 
 
+@pytest.fixture(scope="function")
+def non_internal_user_find_test_client(mocked_pf_auth) -> Generator[FlaskClient, None, None]:
+    """
+    :return: A flask test client authenticated for an external user
+    ie. without @communities.gov.uk or @test-communities.gov.uk email.
+    """
+    app = create_app(config.Config)
+    app.test_client_class = _FindFlaskClient
+    with app.test_client() as test_client:
+        yield test_client
```

Comment:
Where are we specifying the email for this user? Would be goods if they can have a user explicitly auth'd with a non-communities email?

Reply:
Yes it's using the `mocked_pf_auth` fixture which mocks an authorised user with a non-communities email